src/shared/components/drop-target.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+
+const StyledDropTarget = styled.div`
+  align-content: center;
+  align-items: center;
+  background: #ededed;
```

Comment:
I'm wondering if we shouldn't make this a pretty generic container (that is, have the option of changing some of the default styling). This component will likely need to be pretty flexible going forward, so let's consider various options. For instance, let's say we actually add a files drawer (like in vscode / other editors & ides) and we want to make it a drag-and-drop area. What styling choices would we need for this component? It would be great if we could solve this issue in a generalized way once and not have to re-implement it again in the future, so let's think through some ways we can keep it as such.